drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -3114,7 +3114,7 @@ void RasterizerSceneGLES3::_copy_screen(bool p_invalidate_color, bool p_invalida
 
 		GLenum attachments[2] = {
 			GL_COLOR_ATTACHMENT0,
-			GL_DEPTH_STENCIL_ATTACHMENT
+			GL_DEPTH_ATTACHMENT
```

Comment:
RasterizerStorageGLES3::_render_target_allocate allocates depth and stencil, Stencil will likely never be used in GLES3, so feel free to go to that code and ensure stencil is not ever used there too.

Reply:
> RasterizerStorageGLES3::_render_target_allocate allocates depth and stencil, Stencil will likely never be used in GLES3, so feel free to go to that code and ensure stencil is not ever used there too.

I'll do some hunting:)